.circleci/config.yml

```diff
@@ -47,17 +39,11 @@ workflows:
           name: "NixOS 21.11"
           nixpkgs: "21.11"
 
-      # Test against PyPy 2.7
-      - "pypy27-buster":
-          {}
-
-      # Test against Python 3:
-      - "python37":
-          {}
+      # Eventually, test against PyPy 3.8
+      #- "pypy27-buster":
+      #    {}
 
       # Other assorted tasks and configurations
-      - "lint":
-          {}
```

Comment:
Maybe the "codechecks3" CI job and tox environment can be renamed to just "codechecks".